mongodbatlas/containers.go

```diff
@@ -49,17 +49,18 @@ type ContainersListOptions struct {
 
 // Container represents MongoDB network peering containter.
 type Container struct {
-	AtlasCIDRBlock      string `json:"atlasCidrBlock,omitempty"`
-	AzureSubscriptionID string `json:"azureSubscriptionId,omitempty"`
-	GCPProjectID        string `json:"gcpProjectId,omitempty"`
-	ID                  string `json:"id,omitempty"`
-	NetworkName         string `json:"networkName,omitempty"`
-	ProviderName        string `json:"providerName,omitempty"`
-	Provisioned         *bool  `json:"provisioned,omitempty"`
-	Region              string `json:"region,omitempty"`
-	RegionName          string `json:"regionName,omitempty"`
-	VNetName            string `json:"vnetName,omitempty"`
-	VPCID               string `json:"vpcId,omitempty"`
+	AtlasCIDRBlock      string   `json:"atlasCidrBlock,omitempty"`
+	AzureSubscriptionID string   `json:"azureSubscriptionId,omitempty"`
+	GCPProjectID        string   `json:"gcpProjectId,omitempty"`
+	ID                  string   `json:"id,omitempty"`
+	NetworkName         string   `json:"networkName,omitempty"`
+	ProviderName        string   `json:"providerName,omitempty"`
+	Provisioned         *bool    `json:"provisioned,omitempty"`
+	Region              string   `json:"region,omitempty"`     // Region is available for AZURE
+	Regions             []string `json:"regions,omitempty"`    // Regions are available for GCP
+	RegionName          string   `json:"regionName,omitempty"` // RegionName is available for AWS
```

Comment:
I don't understand why we decide to use Region/Regions/RegionName instead of AwsRegion/GcpRegion and AzureRegion 🤷‍♂️